### PR TITLE
Milestone5

### PR DESCRIPTION
**Pull request by shloch**
For code review

Added the event listeners, edited READMe and deployed final milestone to heroku

### CODE REVIEW

File: src/components/Apps.js [hunk 1]
```diff
@@ -1,14 +1,38 @@
...
+    this.setState({
+      total: total,
+      next: next,
+      operation: operation
+    })
+  }
+
   render() {
+    const { total, next } = this.state;
+    //const display = total || next;
```
Comment (SunnyAsar) [inline src/components/Apps.js]: please remove comment

File: src/components/Apps.js [hunk 2]
```diff
@@ -1,14 +1,38 @@
...
+    this.state = {
+      total: null,
+      next: null,
+      operation: null
+    }
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick = (buttonName) => {
+    const { total, next, operation } = calculate(this.state, buttonName);
```
Comment (SunnyAsar) [inline src/components/Apps.js]: TIP:

- when you use arrow function syntax, you don't necessarily need to bind `this` in the constructor.

File: src/logic/operate.js [hunk 3]
```diff
@@ -11,13 +11,11 @@ export default function operate(numbernumber1, numbernumber2, operation) {
...
-      if (number2 === 0) {
+      if (numberTwo === 0)
         alert(`Division by Zero unauthorized`);
-        return;
-      }
-      else {
+      else
         return number1.div(number2).toString();
-      }
+      break;
```
Comment (SunnyAsar) [inline src/logic/operate.js:18]: sorry 😭 but this is still going to break. the button name is a string and using `===` checks for type also so comparing string to integer will fail.
Screenshot 2019-10-24 at 8 32 58 AM



- please fix this.

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hi @shloch 👏 
### You have done very well 🎉 
Unfortunately, we have one deal breaker with the divide function which i have made a comment on .
also remember to remove commented code, we don't need them.
please update this quickly and let us move to the next project.

**Please request another review with the changes in place**

File: src/logic/operate.js (hunk 3 above)
Comment (shloch) [reply src/logic/operate.js:18]: Thank you for this reminder '==='

**Review by SunnyAsar — APPROVED** [review]
# Hi @shloch  😃 

### Amazing! it all works as expected now.
- consider making the display font bigger.
## Congratulation on Finishing the project. let's move to  the final one 🎉 
# You May Pass 👍 
Happy Coding 😁

**Pull request merged**